controllers/internal/ibmblockcsi/default_setter.go

```diff
@@ -82,6 +82,14 @@ func (c *IBMBlockCSI) setDefaults() bool {
 		changed = true
 	}
 
+	if c.Spec.CallHome.Repository != config.DefaultCr.Spec.CallHome.Repository ||
```

Comment:
as discussed, I don't think the default cr has any default values for call home

Reply:
done